module_fcst_grid_comp.F90

```diff
@@ -133,6 +137,173 @@ end subroutine SetServices
 !-----------------------------------------------------------------------
 !#######################################################################
 !-----------------------------------------------------------------------
+!
+  subroutine SetServicesNest(nest, rc)
```

Comment:
I thought you wanted to rename SetSerivcesNest() to something without the word "Nest"? E.g. SetServicesDomain()? Then also rename the variable "nest" to "domain"? It will complicate Dan's merge. But if we don't do it now, it will probably stay the way it is.

Reply:
Let's remember to rename this routine in the following PR (Dan will be updating this routine in his PR anyway).

Reply:
FYI for @danrosen25.